jdaviz/configs/default/plugins/data_quality/tests/test_data_quality.py

```diff
@@ -0,0 +1,76 @@
+import os
+import tempfile
+import pytest
+import numpy as np
+
+from stdatamodels.jwst.datamodels.dqflags import pixel as pixel_jwst
+
+from jdaviz.configs.imviz.plugins.parsers import HAS_ROMAN_DATAMODELS
+from jdaviz.configs.default.plugins.data_quality.dq_utils import (
+    load_flag_map, write_flag_map
+)
+
+
+@pytest.mark.parametrize(
+    "mission_or_instrument, flag, name,",
+    [['jwst', 26, 'OPEN'],
+     ['roman', 26, 'RESERVED_5'],
+     ['jwst', 15, 'TELEGRAPH'],
+     ['roman', 15, 'TELEGRAPH']]
+)
+def test_flag_map_utils(mission_or_instrument, flag, name):
+    flag_map = load_flag_map(mission_or_instrument)
+    assert flag_map[flag]['name'] == name
+
+
+def test_load_write_load():
+    # load the JWST flag map
+    flag_map = load_flag_map('jwst')
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
Just happened to notice it scrolling by. Please use `tmp_path`. Thanks.

https://docs.pytest.org/en/stable/how-to/tmp_path.html

Reply:
Done, thanks.